### DISCUSSION TITLE
More verbose leaderboard

### DISCUSSION THREAD

**Discussion by Simon125q** [dailydotdev/daily#1521] [Feature Requests]
In my opinion it would be really nice to see leaderboard not only of all time but also from the last month, week and maybe day or 48h. In addition to that it would be nice to show which place on leaderboard we are currently on even when we are not in the top 10. I think this would motivate more people to be active and look at the leaderboard as they would have more chance to appear on it.

**Comment by idoshamun** [comment]
Hi @Simon125q,

Thanks for the suggestion! We will consider it :)
